content/post/74-tunneling-http-in-ipfs.md

```diff
@@ -0,0 +1,63 @@
+---
+date: 2019-03-08
+url: 74-tunneling-http-in-ipfs
+tags: libp2p, go-ipfs
+title: Securely tunneling HTTP through IPFS with libp2p
+author: Hector Sanjuan
```

Comment:
@hsanjuan I learned that having an illustration to explain the plumbing wizardry of some of our tech, does wonders when it comes to passing the message through. Wanna take a stab at sketching something?

Reply:
We have diagrams explaining our use of it in several places, which might be useful to link directly to:
https://book.peergos.org/dev/proxy.html
https://peergos.org/blog
https://book.peergos.org/architecture/physical.html